content/architecture/wire-protocol.md

```diff
@@ -20,7 +20,7 @@ The message basic structure is comprised by:
   - Size of the payload.
 - Payload for carrying data. Optional and its size is variable.
 
-You can check the details on `WireProtocol.h](@ src\CLR\WireProtocol\WireProtocol.h)
+You can check the details on `WireProtocol.h`(@ src\CLR\Include\WireProtocol.h)
```

Comment:
```suggestion
You can check the details on [`WireProtocol.h`](https://github.com/nanoframework/nf-interpreter/blob/225581fc4289dcc85add88300c89222ff3919134/src/CLR/Include/WireProtocol.h).
```

Reply:
Updated.

Reply:
Guess one can always switch branches to go to the default one, rigth?
Or maybe just use `https://github.com/nanoframework/nf-interpreter/blob/main/src/CLR/Include/WireProtocol.h` in the URL... I'm fine with either.

Reply:
Modified.
Please check.